### DISCUSSION TITLE
DeepFloyd/IF-I-XL-v1.0

### DISCUSSION THREAD

**Discussion by BennettFourr** [oobabooga/text-generation-webui#1801] [General]
Has anyone found out how to run DeepFloyd/IF-I-XL-v1.0 with textgenwebui.

**Comment by eucdee** [comment]
did you check their [colab](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/deepfloyd_if_free_tier_google_colab.ipynb)?

its a pixel-based text-to-image generation model.
might want to ask here at [automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
but development has slowed down and we maybe have more luck with [automatic](https://github.com/vladmandic/automatic) 
when i first saw IF i was thinking about [ComfyUi](https://github.com/comfyanonymous/ComfyUI)

i guess it will still be a couple of day untill we get a nice ui  ;)  from the colab it doesnt look too complicated...

**Comment by BennettFourr** [comment]
Sorry i ment to post this on auto11
